tests/integration/test_definitions.py

```diff
@@ -23,6 +23,8 @@ def define_tests():
     of (shell command, condition(s) to satisfy).
     """
     celery_regex = r"(srun\s+.*)?celery\s+(-A|--app)\s+merlin\s+worker\s+.*"
```

Comment:
maybe change this to celery_slurm_regex for consistency?

Reply:
OK, I changed that name.